### PR TITLE
Toninittolo, jrr-cpt, MoffMade

### PR DESCRIPTION
**Pull request by ToniNittolo**
Tutorial complete

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -5,25 +5,35 @@
   "requires": true,
   "dependencies": {
     "decktape": {
-      "version": "2.8.3",
```
Comment (bgruening) [inline package-lock.json]: please do not touch those files

**Comment by ToniNittolo** [comment]
That should be taken care of now. Let me know if that is not the case.

Thank you for your time,
     Toni Nittolo
Senior Undergraduate Student
Department of Biochemistry & Biophysics
Texas A&M University, College Station, TX
Phone: (972)741-3007

**Comment by bgruening** [comment]
@ToniNittolo I do not see it :( Also there seems to be a merge conflict. Let us know if you need help here.
Also ping @abretaud and @erasche

**Comment by ToniNittolo** [comment]
I am not sure why my push did not execute, nor do I know what the merge
conflict is.


Senior Undergraduate Student
Department of Biochemistry & Biophysics
Texas A&M University, College Station, TX
Phone: (972)741-3007

**Comment by MoffMade** [comment]
Hey! Looks like this PR went back to Galaxy Main, not our lab fork. I'll get it sorted. We can close this out.

**Pull request closed without merging**